src/Uno.UI.RemoteControl.VS/EntryPoint.cs

```diff
@@ -519,14 +519,11 @@ private async Task CreateInfoBarAsync(NotificationRequestIdeMessage e, IVsShell
 
 		var infoBar = await _infoBarFactory.CreateAsync(
 			new InfoBarModel(
-				e.Message,
-				e.Commands.Select(Commands => new ActionBarItem
-				{
-					Text = Commands.Text,
-					Name = Commands.Name,
-					ActionContext = Commands.Parameter,
-					IsButton = true,
-				}).ToArray(),
+				new ActionBarTextSpan[] {
+					new(e.Title, Bold: true),
```

Comment:
I suggest including a blank space between the message and the title.
Something like this, using a constant.

![image](https://github.com/user-attachments/assets/13a4ab29-f159-4169-8f43-79760ca332a2)

Because it is getting very close and generating noise in the reading.

![image](https://github.com/user-attachments/assets/e53c3bb5-488c-4a85-baef-101faf467420)



The result would be this.

![image](https://github.com/user-attachments/assets/fac5f809-d2b5-4abf-86bc-9de2af7910a5)

Reply:
Makes sense

Reply:
Agreed, this is more appropriate.